src/v3/src/i18n.ts

```diff
@@ -10,23 +10,23 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { initOktaReactI18n } from './lib/okta-i18n';
-import enTranslations from './properties/json/login.json';
+// import { initOktaReactI18n } from './lib/okta-i18n';
```

Comment:
looks like you remove it from the index file, maybe also remove it from jest.setup.js, then you can keep this file unchanged, or just remove it.